Dockerfile

```diff
@@ -14,10 +14,23 @@ COPY ./ /app/
 
 RUN npm run build
 
+#
+# Unprivileged runner stage, runs the application in nginx
+#
+FROM nginxinc/nginx-unprivileged:stable-alpine AS unprivileged
+
+COPY --link --from=builder /app/.docker/nginx.conf  /etc/nginx/conf.d/default.conf
+COPY --link --from=builder /app/dist/ /usr/share/nginx/html/
+
+# set port to >1024 port for non root running
+RUN sed -i 's/80/8080/g' /etc/nginx/conf.d/default.conf
```

Comment:
Maybe move this `sed` call into the builder so you don't need an extra docker layer here?

Reply:
Yes good one, thanks! Done in a new commit